GameServerLib/Logic/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -211,7 +225,15 @@ public Vector2 GetRespawnPosition()
 
         public Spell GetSpell(byte slot)
         {
-            return Spells[slot];
+            if (Spells.ContainsKey(slot))
+            {
+                return Spells[slot];
+            }
+            else
+            {
+                ApiFunctionManager.LogInfo("There is no spell in slot " + slot + ". Using empty spell in its place.");
+                return new Spell(this, "", slot);
```

Comment:
There's exactly what you would want to use as an empty spell: BaseSpell.
Riot uses these for non-champion objects, and I believe going that way is better than using an empty string.